Common/src/dual_grid_structure.cpp

```diff
@@ -59,7 +59,7 @@ CPoint::CPoint(unsigned short val_nDim, unsigned long val_globalindex, CConfig *
 	Coord = new su2double[nDim];
 
 	/*--- Indicator if the control volume has been agglomerated ---*/
-  
+    Parent_CV = 0;
```

Comment:
Yep, I forgot the other 2 constructors..I just fixed them and indentation should be ok as well.
Let me know if we can further improve  the style.